test/unit/model/bulk-create.test.js

```diff
@@ -41,6 +47,16 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           { account_id: 42, id: null },
         ]);
       });
+
+      it('should not return uppercased A', async function () {
```

Comment:
this test doesn't belong inside of the `validations` describe, can you place it outside of that block?

I also would name it more explicitly:

```suggestion
      it('skips setters if the raw option is true', async function () {
```